recipes-kernel/linux/linux-toradex-kmeta.inc

```diff
@@ -0,0 +1,14 @@
+SRCREV_meta = "c6baa89ab5b0a94d177a098f01e3d5e79d57fdf7"
+SRCREV_meta_use-head-next = "${AUTOREV}"
```

Comment:
If i google `SRCREV_meta_use-head-next` i cannot find anything useful. What is that?

Reply:
I don't know. just copied from TDX.

Reply:
It's a 1:1 copy from https://github.com/toradex/meta-toradex-torizon/blob/dunfell-5.x.y/recipes-kernel/linux/linux-toradex-kmeta.inc.

Reply:
Ok. Hope they know what they are doing.